spec.bs

```diff
@@ -1704,7 +1704,8 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   To <dfn>canonicalize a protocol</dfn> given a string |value|:
 
   1. If |value| is the empty string, return |value|.
-  1. Let |dummyURL| be a new [=URL record=].
+  1. Let |dummyInput| be "http://dummy.test".
```

Comment:
It needs to be
```
"`http://...`"
```
so the string value ends up wrapped in a `code` element.

Reply:
I think we should use `https://dummy.invalid/` as URL to more clearly indicate it's not meant to go anywhere and `https` because that's preferred over `http`.

Also, we should introduce "create a dummy URL" in the High-level operations section and invoke that here and elsewhere so the logic is more centralized.

Reply:
Done. I'm a bit unsure where we have the create a dummy URL algorithm. I put it in the High-level operations section, but it might be better to put it in https://urlpattern.spec.whatwg.org/#urlpattern-internals?